packages/core/utils/src/dml/__tests__/entity-builder.spec.ts

```diff
@@ -236,6 +236,9 @@ describe("Entity builder", () => {
       expect(metaData.className).toEqual("User")
       expect(metaData.path).toEqual("User")
 
+      const u = new User()
+      expect(u.username).toEqual(null)
```

Comment:
```suggestion
      expect(new User().username).toEqual(null)
```

just to avoid having a one letter naming here 😄 wdyt?, same comment for the others

Reply:
I have added another assertion that mutates the value. So we will need a variable now 😇

Reply:
Renamed variable to `userInstance`

Reply:
wonderful :)